### PR TITLE
Chore/component

### PR DESCRIPTION
**Pull request by Scrunshes**

### CODE REVIEW

File: src/pagination.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    if (typeof this.pagechange === 'function') {
+      this.pagechange({page:page});
+    }
+    this.page = page;
+
+  }
+
+  updateRecordCount () {
```
Comment (RWOverdijk) [inline src/pagination.js]: I think you have to choose. Either force the user to supply the count, or also offer support for loading data in load (like data-table does). I personally prefer the latter.

**Comment by RWOverdijk** [comment]
Actually starting to look pretty good... I'll try to use this somewhere tonight or tomorrow!

**Comment by doktordirk** [comment]
import directly and not via aurelia-framework (best practise, see other plugins)

**Comment by doktordirk** [comment]
```
if (page === 0) {
      return;
    }
```
seems unsave to me. i'd use <=0 to be on the save side

**Comment by RWOverdijk** [comment]
@doktordirk Comment on the line directly, so we can reply :p page === 0 is good. Be strict :)

**Comment by doktordirk** [comment]
use i18n for 'Next'?
https://github.com/Scrunshes/aurelia-pagination/blob/1858474427400e3f6a2bd4f0233f3e791a1153a3/src/pagination.html#L10

**Comment by Scrunshes** [comment]
Thanks for the feedbacks, i'm commiting changes soon :) Please comment on the line @doktordirk it's easier to read :)

**Comment by Scrunshes** [comment]
If count=0 then then pageCount is gonna be 0 which means there is just gonna be Next and Previous button which is not great, I will be adding pageCount checks so that if it's <= 1 nothing is shown

**Pull request merged**